.circleci/config.yml

```
@@ -37,12 +66,22 @@ jobs:
 
       - run:
           name: Run tests
-          command: npm test
+          command: npm test  
```

yamllint

[error] trailing spaces

(trailing-spaces)